packages/server/utils/analytics/analytics.ts

```diff
@@ -387,6 +388,20 @@ class Analytics {
       newName
     })
 
+  billingLeaderModified = (
+    userId: string,
+    viewerId: string,
+    orgId: string,
+    modificationType: 'add' | 'remove'
```

Comment:
The [issue](https://github.com/ParabolInc/parabol/issues/7813#issuecomment-1495078347) mentioned including a `modificationType` with `add`, `update`, and `remove`, but we're only able to add and remove billing leaders at the moment, so I've included these two options